File: RecyclingApp/app/src/main/java/com/example/recyclingapp/MainActivity.java
package com.example.recyclingapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.content.Intent;
import android.widget.EditText;
import android.widget.ImageSwitcher;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.Toast;
import java.util.Calendar;

public class MainActivity extends AppCompatActivity {



    public void onClickFloatingAction(View buttonView){
        Intent cameraIntent = new Intent(this, cameraActivity.class);
        startActivity(cameraIntent);
    }


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: RecyclerExtreme/app/src/main/java/com/example/recyclerextreme/MainActivity.kt
package com.example.recyclerextreme

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import android.view.View

const val CAMERA_INTENT = "com.example.recyclerextreme.CAMERA"

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onCamera(view: View){
        val camIntent = Intent(this,CameraActivity::class.java)

        startActivity(camIntent)
    }
}
